NTupleAnalyzerHH_nanoAOD.py

```diff
@@ -1914,7 +2166,7 @@ def JERModifiedPt(pt,eta,phi,T,modtype):
 	jet.SetPtEtaPhiM(pt,eta,phi,0.0)
 	for n in range(len(T.GenJet_pt)):
 		gjet = TLorentzVector()
-		gjet.SetPtEtaPhiM(T.GenJet_pt[n],T.GenJet_eta[n],T.GenJet_phi[n],0.0)
+		gjet.SetPtEtaPhiM(T.GenJet_pt[n],T.GenJet_eta[n],T.GenJet_phi[n],T.GenJet_mass[n])
```

Comment:
This looks like a problem in the indentation compared to the old version, please check.